packages/x-charts-pro/src/BarChartPro/BarChartPro.tsx

```diff
@@ -12,7 +12,6 @@ import { ChartsAxisHighlight } from '@mui/x-charts/ChartsAxisHighlight';
 import { ChartsTooltip } from '@mui/x-charts/ChartsTooltip';
 import { ChartsClipPath } from '@mui/x-charts/ChartsClipPath';
 import { useBarChartProps } from '@mui/x-charts/internals';
-import { BarPlotProps } from '@mui/x-charts';
```

Comment:
The lint should catch that one no?

Reply:
I found why (thanks to `npx eslint --print-config [file path]` config.

Our config was such that it prevents you to do a root import of the package itself